### PR TITLE
onSkillUpdated additions

### PR DESCRIPTION
**Pull request by LazzaAU**
This should in theory restore users icon location in My home and merge dialogTemplate files after the skills been updated

### CODE REVIEW

File: HomeAssistant.py [hunk 1]
```diff
@@ -1048,22 +1045,29 @@ def mergeDialogSlots(self):
...
 
-	#onStop. backup display coordinates and dialogTemplate file
+
+	# onStop. backup display coordinates and dialogTemplate file
 	def onStop(self):
 		if self.getConfig('enableBackup'):
-			self.runBackup() #save dialogTemplate file and display settings to backup directory
+			self.runBackup()  # save dialogTemplate file and display settings to backup directory
 		else:
 			self.logInfo(f'![green](Stopped)')
```
Comment (Psychokiller1888) [inline HomeAssistant.py:1068]: def onStop(self):
    if self.getConfig('enableBackup'):
        self.runBackup()
    super.onStop()

**Review by Psychokiller1888 — CHANGES_REQUESTED** [review]

File: HomeAssistant.py (hunk 1 above)
Comment (Psychokiller1888) [reply HomeAssistant.py:1068]: If you don't super call the onStop, the skill is not correctly stopped, intents are not deactivated, skill still appears as active etc etc

Comment (LazzaAU) [reply HomeAssistant.py:1068]: interesting, That would explain why mpbs still had debug messages after uninstalling the skill perhaps.

adding that to pycharm comes back with "unresolved attribute reference to 'onStop' for class 'super'

Comment (Psychokiller1888) [reply HomeAssistant.py:1068]: Should not: https://puu.sh/GwUc3.png

Maybe you haven't configured the project dependency in pycharm?

Comment (LazzaAU) [reply HomeAssistant.py]: ohh crap...
```HomeAssistant]           Stopped
Error while shutting down manager SkillManager: type object 'super' has no attribute 'onStop'
Traceback (most recent call last):
  File "/home/pi/ProjectAlice/core/base/SuperManager.py", line 206, in onStop
    manager.onStop()
  File "/home/pi/ProjectAlice/core/base/SkillManager.py", line 398, in onStop
    skillItem.onStop()
  File "/home/pi/ProjectAlice/skills/HomeAssistant/HomeAssistant.py", line 1067, in onStop
    super.onStop()
AttributeError: type object 'super' has no attribute 'onStop'
```

Comment (Psychokiller1888) [reply HomeAssistant.py]: WTF? Can't be, HomeAssistant class is an AliceSkill

See the o with up arrow? That means it calls the superclass
https://puu.sh/GwUE9.png

Comment (LazzaAU) [reply HomeAssistant.py]: ahh you have super() in that example. that might be the reason , was just super.onStop() previously : )

Comment (Psychokiller1888) [reply HomeAssistant.py]: Lol, must be dislexic today, did not even pay attention to it :)

**Pull request merged**